.github/RELEASE_TEMPLATE.md

```diff
@@ -1,150 +1,89 @@
-# Overview
+Compatible with [`jormungandr@{{JORM_TAG}}`](https://github.com/input-output-hk/jormungandr/releases/tag/{{JORM_TAG}}) and [`cardano-node@{{CARDANO_NODE_TAG}}`](https://github.com/input-output-hk/cardano-node/releases/tag/{{CARDANO_NODE_TAG}}).
```

Comment:
This one feels a bit redundant with the installation instruction below. I'd suggest moving the tags into the title below to make them standing out a bit.

Reply:
It is a bit redundant, but I believed the compatible node versions were important enough to be shown here at the top.

You might be actively seeking a cardano-wallet release based on the node version it supports.